relay-dynamic-config/src/global.rs

```diff
@@ -13,6 +15,28 @@ pub struct GlobalConfig {
     pub measurements: Option<MeasurementsConfig>,
 }
 
+impl GlobalConfig {
+    /// Returns the full filepath of the global config.
+    fn path(base: &Path) -> PathBuf {
+        base.join("global_config.json")
+    }
+
+    /// Loads the [`GlobalConfig`] from a file if it's provided.
+    pub fn load(base: &Path) -> anyhow::Result<Option<Self>> {
+        let path = Self::path(base);
+
+        match path.exists() {
+            true => {
+                let file_contents = std::fs::read_to_string(path)?;
+                let global_config = serde_json::from_str::<GlobalConfig>(file_contents.as_str())?;
```

Comment:
_nit_: There's a way to avoid going through an intermediate string. See the example of [`serde_json::from_reader`](https://docs.rs/serde_json/latest/serde_json/fn.from_reader.html).

```suggestion
    let file = BufReader::new(File::open(path)?);
    let global_config = serde_json::from_reader(file)?;
```